plugins/crate-copy-azure/src/test/java/io/crate/copy/azure/AzureFileOutputTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.copy.azure;
+
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+
+import org.elasticsearch.common.settings.Settings;
+import org.junit.Test;
+
+public class AzureFileOutputTest {
+
+    @Test
+    public void test_reject_unknown_setting() throws Exception {
+        Settings settings = Settings.builder().put("dummy", "dummy").build();
+
+        assertThatThrownBy(() -> new AzureFileOutput(mock(SharedAsyncExecutor.class), settings))
+            .isExactlyInstanceOf(IllegalArgumentException.class)
+            .hasMessage("Setting 'dummy' is not supported");
+    }
```

Comment:
Can be removed if the logic is already tested by a dedicated `AzureBlobStorageSettingsTest`

Reply:
I will remove `openDALConfig` and call `validate` in 2 places (https://github.com/crate/crate/pull/16468#discussion_r1757059322), we would need 2 tests as Input/Output as validating in one place doesn't guarantee that we have validation call in another.